frappe/website/doctype/web_form/web_form.py

```diff
@@ -664,6 +664,11 @@ def get_link_options(web_form_name, doctype, allow_read_on_all_link_options=Fals
 		show_title_field_in_link = (
 			frappe.db.get_value("DocType", doctype, "show_title_field_in_link", cache=1) == 1
```

Comment:
It is already implemented here. Is it not returning correct value?

Reply:
if you coustom frappe app doctype show_title_field_in_link ,it not work!